openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2498,13 +2350,13 @@ The technology described in this specification was made available from contribut
    * changes proof type descriptions to accommodate for the batch issuance changes
    * fix indentation of examples
    * changes proof type descriptions to accomodate for the batch issuance changes
-   * changed Batch Issuance endpoint to differentiate between issuance Credential Configuration, Credential Dataset and instances of Credentials with different key material
```

Comment:
I really don't think anything in the document history should be removed, please rather add one line that you removed batch credential endpoint.

Reply:
As this was a change log entry for the yet-to-be-published draft 14, I believe it is correct to remove it as when we publish -14 it will no longer contain the changes that are being described by this line and hence the only change we need to draw the readers attention to is the removal of the batch endpoint.

(I agree that we should not edit any history entries for published drafts.)

Reply:
makes sense